tools/depcheck/pkg/cmd/pkg.go

```diff
@@ -108,6 +118,43 @@ func BuildGraph(packages *PackageList, excludes []string) (*depgraph.MutableDire
 	return g, nil
 }
 
+// CalculatePackages receives a set of go import paths and runs "go list"
+// using each path as an entrypoint.
+// Returns a PackageList containing dependency and importPath data for each package.
+func CalculatePackages(entrypoints []string) (*PackageList, error) {
```

Comment:
uncalled in this pull?

Reply:
yeah, will be called by the trace and analyze commands.
Can wait to add this as part of the trace cmd if that makes more sense